modules/pensions/lib/pensions/military_information.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+require 'va_profile/prefill/military_information'
+require 'claims_api/service_branch_mapper'
+require 'pensions/form_military_information'
+
+module Pensions
+  ##
+  # Extends MilitaryInformation to add additional prefill methods to Pensions military information prefill
+  # @see lib/va_profile/prefill/military_information.rb VAProfile::Prefill::MilitaryInformation
+  class MilitaryInformation < ::VAProfile::Prefill::MilitaryInformation
```

Comment:
If `lib/va_profile/prefill/military_information.rb` changes, could that impact pensions?

Reply:
@rmtolmach The previous file `lib/pension_21p527ez/pension_military_information.rb` was extending `military_information.rb` already, all i did was move the logic into pensions.